packages/cojson/src/coValues/coMap.ts

```diff
@@ -248,7 +269,7 @@ export class RawCoMapView<
       [K in keyof Shape & string]: Shape[K];
     }> = {};
 
-    for (const key of this.keys()) {
+    for (const key of Object.keys(this.latest) as (keyof Shape & string)[]) {
```

Comment:
Using `Object.keys` instead of `this.keys` because we don't need to filter them by last operation since we use `this.get` internally